### PR TITLE
Sor10874/readmes/leftovers

### PR DESCRIPTION
**Pull request by sorenoid**
these samples got left behind mostly by me. 

create-geometries 
integrated-windows-auth (kotlin)
set-min-max (kotlin)

### CODE REVIEW

File: kotlin/integrated-windows-authentication/README.md [hunk 1]
```diff
@@ -1,32 +1,47 @@
...
-![Integrated windows authentication App](integrated-windows-authentication.png)
+![Image of integrated windows authentication](integrated-windows-authentication.png)
 
 ## Use case
 
-IWA, which is built into Microsoft Internet Information Server (IIS), works well for intranet applications, but isn't always practical for internet apps.
+Your organization might use Integrated Windows Authentication (IWA) to secure ArcGIS Enterprise. This can be useful because the same credentials used to log into your work computer and network can be used to authenticate with ArcGIS. IWA is built into Microsoft Internet Information Server (IIS) and works well for intranet applications but isn't always practical for internet apps.
+
 
 ## How to use the sample
```
Comment (hud10837) [inline kotlin/integrated-windows-authentication/README.md]: I believe this section is not supposed to use the 1.2.3. notation and instead should be a paragraph. You could probably just remove the newlines and numbers

File: kotlin/integrated-windows-authentication/README.md [hunk 2]
```diff
@@ -1,32 +1,47 @@
...
 
-1. Enter the URL to your IWA-secured portal in the edit text view at the top of the screen.
-2. Tap either the "Search Public" (which will search for portals on www.arcgis.com) or "Search Secure" (which will search your IWA-secured portal), for web maps stored on the portal.
-3. If you tap "Search Secure", you will be prompted for a username (including domain, such as username@DOMAIN or domain\username), and password.
-4. If you authenticate successfully, portal item results will display in the recycler view.
-5. Tap a web map item to display it in the map view.
+1. Enter the URL to your IWA-secured portal.
+2. Tap the button to search for web maps stored on the portal.
+3. You will be prompted for a user name, password, and domain (some platforms will use the current Windows login).
```
Comment (hud10837) [inline kotlin/integrated-windows-authentication/README.md]: Perhaps this note about the platforms should be removed or made android specific?

File: kotlin/integrated-windows-authentication/README.md [hunk 3]
```diff
@@ -1,32 +1,47 @@
...
+5. Select a web map item to display it in the map view.
 
 ## How it works
 
-1. The `AuthenticationManager` object is configured with a challenge handler that will prompt for a Windows login (username including domain, and password) if a secure resource is encountered.
+1. The `AuthenticationManager` object is configured with a challenge handler that will prompt for a Windows login (username, password, and domain) if a secure resource is encountered.
 2. When a search for portal items is performed against an IWA-secured portal, the challenge handler creates an `UserCredential` object from the information entered by the user.
-3. If the user authenticates, the search returns a list of web maps from `PortalItem` objects and the user can select one to display as an `ArcGISMap`.
+3. If the user authenticates, the search returns a list of web maps (`PortalItem`) and the user can select one to display as a `Map`.
+4. On some platforms, the current Windows account is used by default and a login prompt will not be shown if it can authenticate successfully.
```
Comment (hud10837) [inline kotlin/integrated-windows-authentication/README.md]: Same here

File: kotlin/integrated-windows-authentication/README.md [hunk 4]
```diff
@@ -1,32 +1,47 @@
-# Integrated Windows Authentication
+# Integrated windows authentication
```
Comment (hud10837) [inline kotlin/integrated-windows-authentication/README.md]: Should windows be capitalized here? Idk.

File: kotlin/integrated-windows-authentication/README.md [hunk 5]
```diff
@@ -1,32 +1,47 @@
...
-3. If you tap "Search Secure", you will be prompted for a username (including domain, such as username@DOMAIN or domain\username), and password.
-4. If you authenticate successfully, portal item results will display in the recycler view.
-5. Tap a web map item to display it in the map view.
+1. Enter the URL to your IWA-secured portal.
+2. Tap the button to search for web maps stored on the portal.
+3. You will be prompted for a user name, password, and domain (some platforms will use the current Windows login).
+4. If you authenticate successfully, portal item results will display in the list.
+5. Select a web map item to display it in the map view.
 
 ## How it works
```
Comment (hud10837) [inline kotlin/integrated-windows-authentication/README.md]: I'm struggling to read the code for this, as I think a lot of stuff happens behind the scenes!

File: kotlin/integrated-windows-authentication/README.md [hunk 6]
```diff
@@ -1,32 +1,47 @@
...
 
-1. The `AuthenticationManager` object is configured with a challenge handler that will prompt for a Windows login (username including domain, and password) if a secure resource is encountered.
+1. The `AuthenticationManager` object is configured with a challenge handler that will prompt for a Windows login (username, password, and domain) if a secure resource is encountered.
 2. When a search for portal items is performed against an IWA-secured portal, the challenge handler creates an `UserCredential` object from the information entered by the user.
-3. If the user authenticates, the search returns a list of web maps from `PortalItem` objects and the user can select one to display as an `ArcGISMap`.
+3. If the user authenticates, the search returns a list of web maps (`PortalItem`) and the user can select one to display as a `Map`.
```
Comment (Rachael-E) [inline kotlin/integrated-windows-authentication/README.md]: update class name here
```suggestion
3. If the user authenticates, the search returns a list of web maps (`PortalItem`) and the user can select one to display as an `ArcGISMap`.
```

File: kotlin/set-min-max-scale/README.md [hunk 7]
```diff
@@ -1,23 +1,30 @@
...
-1. Create an ArcGIS map.
-1. Set min and max scales of map, `ArcGISMap.maxScale = ...` and `ArcGISMap.minScale = ...`.
-1. Set initial Viewpoint of map, `ArcGISMap.initialViewpoint = ...`
-1. Set the ArcGIS map to the `MapView`.
+Zoom in and out of the map. The zoom extents of the map are limited between the given minimum and maximum scales.
+
+## How it works
 
+1. Instantiate an `ArcGISMap` object.
+2. Set min and max scales of map with `arcGisMap.minScale` and `arcGisMap.maxScale`.
```
Comment (Rachael-E) [inline kotlin/set-min-max-scale/README.md]: more common to just refer to the ArcGISMap as "map" for variable use here, or you can use ArcGISMap if you'd prefer.

File: kotlin/integrated-windows-authentication/README.md (hunk 4 above)
Comment (sorenoid) [reply kotlin/integrated-windows-authentication/README.md]: hmmmm. The Microsoft style guide says to capitalize product names "to help people buy them". I'll do it, but under protest.

**Comment by sorenoid** [comment]
@hud10837 I've also ported your IWA suggestions back to the java version, and made your suggested changes. I believe that is it for READMEs.

**Pull request merged**